From 8993e43b8f4d78947db9b18f0e8af804c0f4c4ac Mon Sep 17 00:00:00 2001
From: Nick Clifton <nickc@redhat.com>
Date: Tue, 24 Sep 2019 09:08:00 +0000
Subject: [PATCH] gold: Fix building gold with gcc 10

gold/errors.h uses std::string directly and needs to include <string>
itself for newer compilers.

Upstream-Status: Backport [https://sourceware.org/pipermail/binutils/2019-September/108179.html]
Signed-off-by: Nick Clifton <nickc@redhat.com>

---
 gold/errors.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gold/errors.h b/gold/errors.h
index b4ccd4d722..8f6f486dd4 100644
--- a/gold/errors.h
+++ b/gold/errors.h
@@ -25,6 +25,7 @@
 #include <cstdarg>
 #include <vector>
 #include <cstdio>
+#include <string>
 #include "gold-threads.h"
 
 namespace gold
-- 
2.23.0
